spec/spec_helper.rb

```diff
@@ -19,9 +19,22 @@
   config.run_all_when_everything_filtered = true
   config.raise_errors_for_deprecations!
 
+  config.before(:each) do
+    # clear database
+    ActiveRecord::Base.descendants.each(&:destroy_all)
```

Comment:
Not that it would make a massive difference - but you can just call `delete_all` - which just does a simple `DELETE * FROM <table_name>` compared to the `destroy_all` which calls `destroy` on each object (invoking callbacks).

Reply:
Absolutely right, I didn't notice that :)